packages/react-lowcode/src/codegen/index.ts

```diff
@@ -37,7 +37,7 @@ export function generatePages(inputSourceCode: string, io: CodeRW & CodeDir, opt
             let context = {uiFramework: UiFramework.MaterialUI, formatter: Formatter.None, index: {tableType: TableType.BasicTable, height: "400px"}};
             
             const generator = new AppGenerator(context, entity)
-            const page = generator.generateIndexPage(/* TODO entity / type name should be input - not in context */)
+            const page = generator.generateListComponent(/* TODO entity / type name should be input - not in context */)
```

Comment:
We need to generate both both (two files actually):
* `generateListComponent()`
* `generateListPage()`